front/src/modules/ui/icon/index.ts

```diff
@@ -22,7 +22,6 @@ export {
   IconChevronLeft,
   IconChevronsRight,
   IconCircleDot,
-  IconCirclePlus,
```

Comment:
Not used

Reply:
Let's leave it as it's part of the "authorized/vetted icons" we want developers to be able to use